lib/commands/app/create.ts

```diff
@@ -60,6 +68,11 @@ export default class AppCreateCmd extends Command {
 	public static usage = 'app create <name>';
 
 	public static flags: flags.Input<FlagsDef> = {
+		organization: flags.string({
+			char: 'o',
+			description:
+				'the handle of organization the application should belong to',
```

Comment:
`of organization` → `of the organization`

Reply:
(I also removed the initial unneeded 'the', matching other flag)